.github/workflows/latest_import.yml

```diff
@@ -0,0 +1,129 @@
+name: Weekly late importers
+
+on:
+  schedule:
+    - cron: '0 4 * * 1'  # Run at 04:00 UTC every Monday
```

Comment:
Same schedule as the Technical Debts Counter: is this ok?

Reply:
It should be fine, the Technical Debts Counter doesn't use a Hoskinson runner and also takes much less time than this job anyways.